apparmor.d/groups/full-policy/full-policy

```diff
@@ -0,0 +1,251 @@
+# full-apparmor-policy
+# Full System MAC Policy using AppArmor
+#
+# Copyright (c) 2023 monsieuremre <https://github.com/monsieuremre>
+#  
+# This file is part of full-apparmor-policy. You can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+abi <abi/3.0>,
```

Comment:
Nitpick: add a new line after the abi definitions.

Reply:
Sure.